tests/Unit/Elliptic/Systems/SelfForce/Scalar/BoundaryConditions/Test_Sommerfeld.cpp

```diff
@@ -0,0 +1,62 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cstddef>
+#include <limits>
+#include <string>
+#include <utility>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Structure/Direction.hpp"
+#include "Domain/Tags.hpp"
+#include "Domain/Tags/FaceNormal.hpp"
+#include "Elliptic/BoundaryConditions/ApplyBoundaryCondition.hpp"
+#include "Elliptic/BoundaryConditions/BoundaryCondition.hpp"
+#include "Elliptic/BoundaryConditions/BoundaryConditionType.hpp"
+#include "Elliptic/Systems/SelfForce/Scalar/BoundaryConditions/Sommerfeld.hpp"
+#include "Framework/CheckWithRandomValues.hpp"
+#include "Framework/Pypp.hpp"
+#include "Framework/SetupLocalPythonEnvironment.hpp"
+#include "Framework/TestCreation.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace ScalarSelfForce::BoundaryConditions {
+
+SPECTRE_TEST_CASE("Unit.ScalarSelfForce.BoundaryConditions.Sommerfeld",
+                  "[Unit][Elliptic]") {
+  // Test factory-creation
+  const auto created = TestHelpers::test_factory_creation<
+      elliptic::BoundaryConditions::BoundaryCondition<2>, Sommerfeld>(
+      "Sommerfeld:\n"
+      "  BlackHoleMass: 1.0\n"
+      "  BlackHoleSpin: 0.5\n"
+      "  OrbitalRadius: 10.0\n"
+      "  MModeNumber: 2\n");
+  REQUIRE(dynamic_cast<const Sommerfeld*>(created.get()) != nullptr);
+  const auto& boundary_condition = dynamic_cast<const Sommerfeld&>(*created);
+  {
+    INFO("Semantics");
+    test_serialization(boundary_condition);
+    test_copy_semantics(boundary_condition);
+    auto move_boundary_condition = boundary_condition;
+    test_move_semantics(std::move(move_boundary_condition), boundary_condition);
+  }
+  {
+    INFO("Properties");
+    CHECK(boundary_condition.black_hole_mass() == 1.0);
+    CHECK(boundary_condition.black_hole_spin() == 0.5);
+    CHECK(boundary_condition.orbital_radius() == 10.0);
+    CHECK(boundary_condition.m_mode_number() == 2);
+    CHECK(boundary_condition.boundary_condition_types() ==
+          std::vector<elliptic::BoundaryConditionType>{
+              elliptic::BoundaryConditionType::Neumann});
+  }
```

Comment:
Maybe I'm missing it, but should there be a test that you can invoke the boundary condition?